### DISCUSSION TITLE
[@types/node] Does  gives this error when deploying to onrender

### DISCUSSION THREAD

**Discussion by defiboyszn** [DefinitelyTyped/DefinitelyTyped#63755] [Issues with a @types package]
I don't know why it's giving error

i'm trying to deploy a typescript backend code to onrender but it's failing 

this is the error
 node_modules/@types/node/globals.d.ts(72,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'AbortSignal' must be of type '{ new (): AbortSignal; prototype: AbortSignal; abort(reason?: any): AbortSignal; timeout(milliseconds: number): AbortSignal; }', but here has type '{ new (): AbortSignal; prototype: AbortSignal; }'.

[a comment by an automated account is not shown]

**Reply by peterblazejewicz** [reply]
@Tobithedev there could be different way to fix/solve the problem.
The problem appears when non-compatible types versions are used in mix of lib.dom and types/node. That's unfortunate outcome of the split of JS/Node features.
Similar:
https://github.com/microsoft/TypeScript/issues/51567
please try to add @types/node to your project devDependencies to force proper resolution for your TS version

**Comment by defiboyszn** [comment]
@types/node is in devDependencies in my package.json file
